oshc/main/templates/index.html

```diff
@@ -1,60 +1,106 @@
 {% extends 'base.html' %}
 {% load static %}
 {% block content %}
-  <div class="container">
-    <div class="jumbotron text-center">
-		<img id="logo" src="{% static "main/images/oshc-logo.png"%}" alt="" >
-        <h1>Open Source Help Community</h1>
-        <h3>A Virtual Platform To Help People To Start Contributing To Open Source Projects.</h3>
-    </div>
-  </div>
+	<section class="main-section">
+		<div class="container">
+			<div class="jumbotron text-center">
+				<img class="logo-image" src="{% static "main/images/oshc-logo.png"%}" alt="Open Source Help Community Logo"/>
+				<h1>Open Source Help Community</h1>
+				<h4>A Virtual Platform To Help People To Start Contributing To Open Source Projects</h4>
+			</div>
+		</div>
+	</section>
 
+	<section class="section-two">
+		<div class="container">
+			<h3>Our Mission</h3>
+			<p class="bi">
+				<b>"</b>Open Source Help Community is an a virtual platform to help people to start contributing to open source projects<b>"</b>
+			</p>
+			<p class="intro">
+				OSHC is established with the idea in mind to reach those people who are interested in contributing to open source projects but find it difficult. OSHC organizes sessions with open source projects contributors and publishes material which will be useful for people to get started with contributing to open source projects.
+			</p>
+			<div class="line-gap-10px"></div>
+			<p class="intro">
+				If you would like to share your thoughts or your journey with us, have ideas to make this great, or have questions, please get in touch at <a href="mailto:opensourcehelpcommunity@gmail.com">opensourcehelpcommunity@gmail.com</a>
+			</p>
+		</div>
+	</section>
 
-  <div class="container marketing">
+	<section class="section-three">
+		<div class="container">
+			<h3>Upcoming Sessions</h3>
 
-    <div class="row">
+			<div class="row">
+				<div class="col-md-7 left-session-info">
+					<div class="panel panel-default">
+						<div class="panel-heading">
+							<h3 class="panel-title">OSC Chat Session with Himanshu Mishra</h3>
+						</div>
+						<div class="panel-body">
+							<a href="https://orkohunter.net">Himanshu</a> is a pre-final year undergraduate student at IIT Kharagpur. He has been a Google Summer of Code student in 2015 and 2016 and is currently a mentor in Python Software Foundation.
+							<div class="line-gap-10px"></div>
+							<i>Date: Wed, June 21, 2017 8:30 PM – 9:30 PM IST</i>
+							<div class="line-gap-10px"></div>
+							<button type="button" class="btn btn-lg btn-primary sRegister">
+								<a href="https://www.eventbrite.com/e/open-source-contributors-chat-session-with-himanshu-mishra-tickets-35227478336?ref=ecal">Register Now!</a>
+							</button>
```

Comment:
@amarlearning The new design looks great! 👍 But after this change one has to go and edit the code for new sessions and also remove the previous ones, isn't it? This adds to extra effort before every new session, do you have any ideas for this?